fastlane/Fastfile

```diff
@@ -148,7 +148,10 @@ end
   desc "Removes all the temp tags and puts the final one"
   lane :finalize_release do | options |
     ios_finalize_prechecks(options)
-    ios_update_metadata(options) unless ios_current_branch_is_hotfix
+    unless ios_current_branch_is_hotfix
+      ios_update_metadata(options)
+      ios_bump_version_beta()
+    end
```

Comment:
I snuck this one in because it's release-tooling specific. It makes sure the build number is updated _automatically_. About time, hey?! 😅